.projen/tasks.json

```diff
@@ -198,6 +198,22 @@
           "exec": "react-scripts eject"
         }
       ]
+    },
+    "storybook": {
+      "name": "storybook",
+      "steps": [
+        {
+          "exec": "start-storybook -p 6006 -s public"
+        }
+      ]
+    },
+    "build-storybook": {
```

Comment:
1. Can you add descriptions to the tasks?
2. Maybe call this one `storybook:build` (no strong opinion)